lib/modules/dosomething/dosomething_user/dosomething_user.module

```diff
@@ -429,11 +433,55 @@ function dosomething_user_validate_address_field($form, &$form_state) {
     }
     // We have a full address, save it!
     else {
-      dosomething_user_set_address_values($form, $form_state, $formatted_address);
+      if ($form['#form_id'] === 'dosomething_user_info_form') {
+        dosomething_user_set_address_values_from_info_form($form, $form_state, $formatted_address);
+      }
+      else {
+        dosomething_user_set_address_values_from_edit_form($form, $form_state, $formatted_address);
+      }
     }
```

Comment:
@chloealee I'm having a hard time reading this diff, but this seems like a confusing approach. 
You should be able to leave the `address_field` validation the way it was as that sets and validates correctly 

Since the info form is a different type of field that we build manually you can use either the [menu hook](https://github.com/DoSomething/phoenix/blob/dev/lib/modules/dosomething/dosomething_user/dosomething_user.module#L65) to validate the way [JS does](https://github.com/DoSomething/phoenix/blob/5896b2823f770c6aa3a1e9cf46d474ef4ae60564/lib/themes/dosomething/paraneue_dosomething/js/validators/address.js#L125), or create a new validation to handle the manually created fields that sets form errors and saves the fields

but you shouldn't touch